src/NUnitTestExtractor/NUnitTestExtractor/Options.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using CommandLine;
+namespace NUnitTestExtractor
+{
+    public sealed class Options
+    {
+        [Option('h', "help", HelpText = "Prints this help, to specify DLL files just list them out at the end with spaces inbetween", Required = false, DefaultValue = false)]
+        public bool Help { get; set; }
+
+        [Option('l', "level", HelpText = "Specifies the level at which the test is located", Required = true)]
```

Comment:
Instead of "Specifies the level at which the test is located" maybe something like this would be better:
"Specifies the level of granularity to use for the output tests.  Valid values are: 'namespace', 'class' or 'function')"